project.clj

```diff
@@ -6,14 +9,32 @@
 
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [org.clojure/tools.reader "0.8.9"]
+
                  [ring/ring-core "1.3.2"]
+                 [clj-time "0.7.0"]
                  [commons-io "2.4"]
+                 [commons-codec "1.9"]
+                 [slingshot "0.12.2"]
+                 [org.slf4j/slf4j-api "1.7.7"]
+
                  [bidi "1.18.9" :exclusions [org.clojure/clojurescript]]
                  [compojure "1.3.3"]
                  [prismatic/schema "0.4.0"]
-                 [puppetlabs/kitchensink "1.1.0"]]
+                 [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/trapperkeeper-metrics "0.1.1-SNAPSHOT"]]
```

Comment:
Totally agree that it would be nice for non-PL users to not have to have a dependency on TK, I just understand it we don't want to prioritize that right now.